### DISCUSSION TITLE
fatal: repository 'https://github.com/ltdrdata/ComfyUI-M/' not found

### DISCUSSION THREAD

**Discussion by Method237** [community/community#121195] [New to GitHub]
Hello I need help with this. I am trying to clone my ComfyUI file and I get not found responds likr this.

 Cloning into 'ComfyUI-M'...
info: please complete authentication in your browser...
remote: Repository not found.
fatal: repository 'https://github.com/ltdrdata/ComfyUI-M/' not found

**Comment by gurjeetsinghvirdee** [comment]
Hey @Method237,

The link you provided doesn't lead to a repo like that; I think you mistyped the URL. The correct one is this: https://github.com/ltdrdata/ComfyUI-Manager.

First of all, before cloning any repo, you have to fork it. By clicking on this **fork** button, it will make a fresh copy for you: 
![image](https://github.com/community/community/assets/73753957/ac254d66-8675-4d9f-bcf2-84742550b414)

Once you've forked the repo, you can clone your forked repo. Now it will clone to your system without giving any fatal errors.
